### ISSUE TITLE
maxTargets

### ISSUE THREAD

**Issue by praad** [simialbi/yii2-elfinder#2]
Is it possible to override maxTargets by modul config array?
`'modules' => [
        'elfinder' => [
            'class' => 'simialbi\yii2\elfinder\Module',
            'connectionSets' => [
                'default' => [ // like elfinder roots
                    [
                        'class' => 'simialbi\yii2\elfinder\ElFinderConfigurationLocalFileSystem',
                        'path' => '...',
                        'URL' => '..,
                        //'maxTargets' => 1,   // NOT WORKING HERE
                        'attributes' => [`

**Comment by simialbi** [comment]
Should be fixed with newset commit [9445996](https://github.com/simialbi/yii2-elfinder/commit/9445996221634d25f274130f8c6cf8b3bf970450). I'm gonna test it an will publish a stable release.

**Comment by simialbi** [comment]
Fixed with [1.6.0 release](https://github.com/simialbi/yii2-elfinder/releases/tag/1.6.0)

Use it like this:
```php
[
    'modules' => [
        'elfinder' => [
            'class' => 'simialbi\yii2\elfinder\Module',
            'options' => [
                'default' => [
                    'class' => 'simialbi\yii2\elfinder\ElFinderOptions',
                    'maxTargets' => 1500 // whatever you want to set here
                ]
            ],
            'connectionSets' => [
                'default' => [...]
            ]
        ]
    ]
]
```
